Automattic/wp-calypso
# Add email stats for devices & clients

#### Proposed Changes

This PR adds the data logic & UI for the devices & clients panels in email open stats.

In detail, it:

* Adds the Redux logic (routing/transforming the incoming data from the API to the store)
* Adds the UI for the panel
* Adds a grid layout for email open stats
* Adds titles to the panels
* Does a bit of cleanup:
  - Use `<StatsEmailModule>` directly, instead of creating a separate proxy component for every stat type
  - Remove the preview code from the `<StatsEmailOpenDetail>` module


#### Testing Instructions

* Visit `/stats/email/open/<site>/day/<postid>?flags=newsletter/stats`. You can add the startDate parameter, to load the stats for a specific date.
* Ensure the cards are rendered:

![CleanShot 2023-01-04 at 14 28 47@2x](https://user-images.githubusercontent.com/528287/210565507-6f3acce5-d128-4fa7-9c87-aa4b496968f9.png)


#### Pre-merge Checklist





Related to #
